### PR TITLE
Update style guide, quotation marks, emphasis, terminology

### PR DESCRIPTION
**Pull request by dariavladykina**
This update addresses issues #231, #232, #91, #141

### CODE REVIEW

File: xml/docu_styleguide.language.xml [hunk 1]
```diff
@@ -422,16 +422,26 @@
...
   See also <xref linkend="sec-tone"/>.
   </para>
-  </listitem>
+  </listitem>  
   </itemizedlist>
   <para>
   To create quotations, use the <tag class="emptytag">quote</tag>
   element. This element is easier to localize than hardcoded quotation marks.
   During processing, localized quotation marks are added automatically.
+  Avoid using single quotation marks.
```
Comment (tomschr) [inline xml/docu_styleguide.language.xml:444]: Is this a thing? If we use the `<quote>` tag, this won't happen. Sounds a bit unnecessary.

**Review by tomschr — COMMENTED** [review]
Thanks Daria. Just a minor question, otherwise perfect. :+1:

File: xml/docu_styleguide.language.xml (hunk 1 above)
Comment (dariavladykina) [reply xml/docu_styleguide.language.xml:444]: I think we need to discourage the use of single quote marks like 'this'. We mention that hardcoded, i.e. hand-written double marks should not be used, so I believe it's worth mentioning the second type, too.

Comment (tomschr) [reply xml/docu_styleguide.language.xml:444]: Ok, yes, that's sounds reasonable. Then keep it. :slightly_smiling_face:

**Review by janajaeger — APPROVED** [review]

**Review by PadraigDillon1 — APPROVED** [review]

**Review by tomschr — APPROVED** [review]

**Pull request merged**